routes/event/event.js

```diff
@@ -18,12 +26,7 @@ router.get('/create', function (req, res, next) {
         res.redirect('../login');
         return;
     }
-    if (req.session.uid != 1) {
-        res.redirect('../');
-        return;
-    }
-    //Not quite sure what is the part inside render
-    res.render('event/create', {});
+    res.render('event/create', {uid: req.session.uid});
```

Comment:
Make sure to add Auth service to all functions and make sure only the right people have access (eg. create, edit, delete) merge down from master if needed